backend/pretest-script.sh

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# set node env to test, if windows use set instead of export
+if [[ "$OSTYPE" == "msys" ]]; then
+  set NODE_ENV=test
```

Shellcheck (0.10.0)

[warning] To assign a variable, use just 'var=value', no 'set ..'.

(SC2121)